core/src/main/java/com/bumble/appyx/core/node/Node.kt

```diff
@@ -80,6 +84,13 @@ abstract class Node(
         });
     }
 
+    protected suspend inline fun <reified T> executeWorkflow(
```

Comment:
How does this method relate to `attachWorkflow`? Workflow seems to be used in attachWorkflow as another way of adding multiple children (like a deeplink)

I'm struggling to see how they relate other than at what point in the code this can execute (as you said `Workflows can be executed only after root node onBuilt has been executed - when childNodeLifecycleManager.launch() is called`)

Reply:
these 2 methods are independent:
1. `attachWorkflow` can be only called on a parent node as it attaches a child node
2. `executeWorkflow` just performs an action.

We only need to wait for `RootNode` to be built in order to execute a workflow. It doesn't work as is because at the moment there's no callback from composition to know when the root node has been actually composed and built

Reply:
Naming is matching [RIBs](https://github.com/badoo/RIBs/blob/master/libraries/rib-rx2/src/main/java/com/badoo/ribs/rx2/workflows/RxWorkflowNode.kt)